tests/framework/results/test_interface.py

```diff
@@ -91,9 +91,13 @@ def test_register_observation(
     assert len(interface._manager._results_context.observations) == 1
 
 
-def test_register_observations():
+def test_register_observations(mocker):
     mgr = ResultsManager()
     interface = ResultsInterface(mgr)
+    builder = mocker.Mock()
+    builder.configuration.stratification.default = []
+    mgr.setup(builder)
+
```

Comment:
Why did you add this mock and setup() to these tests?

Reply:
Curious as well.

Reply:
The test fails without `setup` being run, as the `logger` doesn't get created without that. When `setup` is run it overrides the default stratification with a Mock object rather than the empty list it was before.